publish.yml

```
@@ -0,0 +1,53 @@
+# Use this playbook file when you need to build and publish multuple versions of your content together
+site:
+  title: Neo4j Docs Template
+  url: https://neo4j.com/docs/add-on
+  start_page: docs-template:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ./
+    branches: ['1.0', 'dev']
+    exclude:
+    - '!**/_includes/*'
+    - '!**/readme.adoc'
+    - '!**/README.adoc'
+
+ui:
+  bundle:
+    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    snapshot: true
+  output_dir: /assets
+
+urls:
+  html_extension_style: indexify
+
+antora:
+  extensions:
+  - require: "@neo4j-antora/antora-modify-sitemaps"
+    sitemap_version: '1.0'
+    sitemap_loc_version: 'current'
+    move_sitemaps_to_components: true
+
+asciidoc:
+  extensions:
+  - "@neo4j-documentation/remote-include"
+  - "@neo4j-documentation/macros"
+  - "@neo4j-antora/antora-add-notes"
+  - "@neo4j-antora/antora-table-footnotes"
+  attributes:
+    page-theme: docs
+    page-type: Docs
+    page-search-type: Docs
+    page-search-site: Reference Docs
+    page-canonical-root: /docs
+    page-pagination: true
+    page-no-canonical: true
+    page-origin-private: true
+    page-hide-toc: false
+    page-mixpanel: 4bfb2414ab973c741b6f067bf06d5575
+    includePDF: false
+    nonhtmloutput: ""
+    experimental: ''
+    copyright: 2022
+    common-license-page-uri: https://neo4j.com/docs/license/
```

```suggestion
    common-license-page-uri: https://neo4j.com/docs/license/
    check-mark: icon:check[]
```

Looks like the template is missing the `check-mark` attribute